velox/exec/tests/utils/AssertQueryBuilder.h

```diff
@@ -160,39 +160,49 @@ class AssertQueryBuilder {
   /// duckDbQueryRunner to be provided in the constructor.
   std::shared_ptr<Task> assertResults(
       const std::string& duckDbSql,
-      const std::optional<std::vector<uint32_t>>& sortingKeys = std::nullopt);
+      const std::optional<std::vector<uint32_t>>& sortingKeys = std::nullopt,
```

Comment:
since this uses the= builder pattern, would it be cleaner to have addSplitWithSequence as a separate API, instead of carrying that as parameter all over? Something like:

```
AssertQueryBuilder().addSplitsWithSequence().assertResults(expected);
```

?

Reply:
Hi @pedroerp, thank you for the suggestion. I've updated the PR. Could you take another look?